frontend/src/app/App.tsx

```diff
@@ -186,6 +187,16 @@ const App = () => {
                 />
               }
             />
+            <Route
+              path="results/upload/submissions/:id"
```

Comment:
One thing that I just noticed is that we're inconsistent here with our pathing - upload is singular submissions plural.  I think it makes sense to have these be consistent

Reply:
Yeah the naming is a bit strange here 🤔 "uploads" and "submissions" would be redundant as they refer to the same resource so I'm not sure what the neatest way to resolve would be... 

I guess in retrospect we could have done `POST /results/submissions` for the upload and `GET /results/submissions` and `GET /results/submissions:id`; that might have been the REST-iest option.